src/lib/operators/print.cpp

```diff
@@ -171,22 +173,25 @@ std::shared_ptr<const Table> Print::_on_execute() {
 // `min` and `max` can be used to limit the width of the columns - however, every column fits at least the column's name
 std::vector<uint16_t> Print::_column_string_widths(uint16_t min, uint16_t max,
                                                    const std::shared_ptr<const Table>& table) const {
-  std::vector<uint16_t> widths(table->column_count());
+  const auto column_count = table->column_count();
+  std::vector<uint16_t> widths(column_count);
```

Comment:
auto-to-stick, curly braces?
```suggestion
  auto widths = std::vector<uint16_t>{column_count};
```